core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -51,7 +51,8 @@
 #include <impl/Kokkos_Tags.hpp>
 #include <impl/Kokkos_AnalyzePolicy.hpp>
 #include <Kokkos_Concepts.hpp>
-#include <iostream>
+#include <iosfwd>
```

Comment:
Actually I am tempted to just remove the header because I don't see where it is being used.

Reply:
I think that's fair. It seems like we don't need that header in a lot of places we are using it right now. Users should not rely on us including headers they need.